LICENSE-MIT

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Traverse Research
+Copyright (c) 2019 Graham Wihlidal
```

Comment:
I thought this file only got renamed when replicating the setup from our intel-tex repo... : Facepalm: